packages/canary/src/components/icon.tsx

```diff
@@ -181,7 +183,9 @@ const IconNameMap = {
   run: Run,
   'run-test': RunTest,
   'ssh-key': SshKey,
-  'green-dot': GreenCircle
+  'green-dot': GreenCircle,
+  'green-tick': GreenTick,
```

Comment:
ideally we should have one tick mark icon, and set color via css/props. instead of separate colored icons.